packages/core/src/components/popover/popoverSharedProps.ts

```diff
@@ -90,6 +90,11 @@ export interface PopoverSharedProps<TProps extends DefaultPopoverTargetHTMLProps
     /** Interactive element which will trigger the popover. */
     children?: React.ReactNode;
 
+    /**
+     * The content displayed inside the popover.
+     */
+    content?: string | React.JSX.Element;
```

Comment:
this shouldn't happen yet, tooltip does nothing with this prop but this would document it

Reply:
tooltip absolutely uses the `content` prop

Reply:
yes seeing that now - didn't see anything removed from tooltip in this diff which threw me off

why are we moving this to shared props?

Reply:
Because it _is_ shared.

Reply:
`TooltipProps` defines its own `content` prop to make it required, but having it in `PopoverSharedProps` ensures that it's the same type. Which makes sense, because `Tooltip` is really just a fancy wrapper for a `Popover`.

 `content` truly is a shared prop between the two, so it makes sense to have it in `PopoverSharedProps`.

Reply:
makes sense to me - thanks for explaining!